bundles/org.openhab.binding.gardena/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -10,1162 +10,633 @@
-		<label>Brightness</label>
-		<description>The brightness</description>
-		<state pattern="%d lx" readOnly="true"/>
-	</channel-type>
-
-	<!-- ############################### Pressure Pump ############################### -->
-
-	<thing-type id="electronic_pressure_pump">
-		<supported-bridge-type-refs>
-			<bridge-type-ref id="account"/>
-		</supported-bridge-type-refs>
-
-		<label>Gardena Smart Pressure Pump</label>
-		<description>Represents a Gardena Smart Pressure Pump</description>
-
-		<channel-groups>
-			<channel-group id="device_info" typeId="deviceInfo"/>
-			<channel-group id="radio" typeId="radio"/>
-
-			<channel-group id="outlet_temperature" typeId="outletTemperature"/>
-			<channel-group id="pump" typeId="pump"/>
-			<channel-group id="outlet_pressure" typeId="outletPressure"/>
-			<channel-group id="flow" typeId="flow"/>
-			<channel-group id="manual_watering" typeId="manualWatering"/>
-			<channel-group id="scheduling" typeId="scheduling"/>
-		</channel-groups>
-
-		<properties>
-			<!-- device info -->
-			<property name="manufacturer"/>
-			<property name="product"/>
-			<property name="serialNumber"/>
-			<property name="sgtin"/>
-			<property name="version"/>
-			<property name="category"/>
-		</properties>
-
-		<config-description>
-			<parameter name="leakage_detection" type="text">
-				<label>Leakage Detection</label>
-				<options>
-					<option value="watering">Watering</option>
-					<option value="washing_machine">Washing machine</option>
-					<option value="domestic_water_supply">Domestic water supply</option>
-					<option value="off">Off</option>
-				</options>
-			</parameter>
-			<parameter name="operating_mode" type="text">
-				<label>Operating Mode</label>
-				<options>
-					<option value="scheduled">Scheduled</option>
-					<option value="automatic">Automatic</option>
-				</options>
-			</parameter>
-			<parameter name="turn_on_pressure" type="text">
-				<label>Turn On Pressure</label>
-				<options>
-					<option value="2.0">2.0</option>
-					<option value="2.2">2.2</option>
-					<option value="2.4">2.4</option>
-					<option value="2.6">2.6</option>
-					<option value="2.8">2.8</option>
-				</options>
-			</parameter>
-		</config-description>
-	</thing-type>
-
-	<channel-group-type id="outletTemperature">
-		<label>Outlet Temperature</label>
-		<description>The outlet temperature</description>
+	<channel-group-type id="mowerCommands">
+		<label>Mower Commands</label>
+		<description>Commands to control a mower</description>
 		<channels>
-			<channel id="temperature" typeId="temperature"/>
-			<channel id="temperature_min" typeId="temperatureMin"/>
-			<channel id="temperature_max" typeId="temperatureMax"/>
-			<channel id="frost_warning" typeId="frostWarning"/>
+			<channel id="commandDuration" typeId="mowerCommandDuration">
+				<label>Mowing Duration</label>
+			</channel>
+			<channel id="start_seconds_to_override" typeId="gardenaCommand">
+				<label>Start mowing with duration</label>
```

Comment:
Labels are expected to be as short as possible. Guideline is 2-3 words with up to 25 chars. See https://www.openhab.org/docs/developer/bindings/thing-xml.html#formatting-labels-and-descriptions